tests/python/pants_test/init/test_logging.py

```diff
@@ -4,63 +4,65 @@
 import logging
 from pathlib import Path
 
-from pants.engine.internals.native import Native
-from pants.init.logging import setup_logging_to_file
+from pants.engine.internals import native_engine
+from pants.init.logging import initialize_stdio
+from pants.testutil.option_util import create_options_bootstrapper
 from pants.util.contextutil import temporary_dir
 from pants.util.logging import LogLevel
 
 
 def test_file_logging() -> None:
-    native = Native()
-    native.init_rust_logging(
-        level=LogLevel.INFO.level,  # Tests assume a log level of INFO
-        log_show_rust_3rdparty=False,
-        use_color=False,
-        show_target=False,
-        log_levels_by_target={},
-        message_regex_filters=(),
-    )
-    logger = logging.getLogger("my_file_logger")
     with temporary_dir() as tmpdir:
-        setup_logging_to_file(LogLevel.INFO, log_dir=tmpdir)
-        log_file = Path(tmpdir, "pants.log")
+        ob = create_options_bootstrapper([f"--pants-workdir={tmpdir}"])
 
-        cat = "🐈"
-        logger.warning("this is a warning")
-        logger.info("this is some info")
-        logger.debug("this is some debug info")
-        logger.info(f"unicode: {cat}")
+        # Do not set up a stdio destination, meaning that all messages will go to the log.
+        global_bootstrap_options = ob.bootstrap_options.for_global_scope()
+        with initialize_stdio(global_bootstrap_options):
+            logger = logging.getLogger(None)
 
-        loglines = log_file.read_text().splitlines()
-        print(loglines)
-        assert len(loglines) == 3
-        assert "[WARN] this is a warning" in loglines[0]
-        assert "[INFO] this is some info" in loglines[1]
-        assert f"[INFO] unicode: {cat}" in loglines[2]
+            cat = "🐈"
```

Comment:
Nit: I'm not sure there's any reason to create a separate variable for this unicode cat emoji and interpolate

Reply:
This was here before, will preserve.